libkbfs/kbfs_ops.go

```diff
@@ -160,7 +160,12 @@ func (fs *KBFSOpsStandard) DeleteFavorite(ctx context.Context,
 		return fs.opsByFav[fav]
 	}()
 	if ops != nil {
-		return ops.deleteFromFavorites(ctx, fs.favs)
+		err := ops.deleteFromFavorites(ctx, fs.favs)
+		if _, ok := err.(OpsCantHandleFavorite); !ok {
```

Comment:
Good question.  But actually either way the request will go to the server.  The only reason we prefer to go through `folderBranchOps` is because there might be some local state to clean up in that object when it's removed from favorites.  (Right now there isn't any, but was trying to plan ahead.)  I believe the different ways to delete favorites are equivalent.